Source/WebCore/platform/graphics/avfoundation/objc/MediaPlayerPrivateAVFoundationObjC.mm

```diff
@@ -1255,12 +1244,14 @@ static URL conformFragmentIdentifierForURL(const URL& url)
     m_haveCheckedPlayability = true;
 
     INFO_LOG(LOGIDENTIFIER);
-    WeakPtr weakThis { *this };
+    __block WeakPtr weakThis { *this };
```

Comment:
Nit: should be able to just create the WeakPtr in the capture list (i.e. `weakThis = WeakPtr { *this }`)

Reply:
I don't think you can do that; the outer lambda is an Objective-c block, not a C++ lambda.

Reply:
Oops, I overlooked the outer block.